src/fourth-iteration.md

```diff
@@ -18,7 +18,7 @@ impl<T> List<T> {
 
 impl<T> Iterator for IntoIter<T> {
     type Item = T;
-    fn next(&mut self) -> Option<T> {
+    fn next(&mut self) -> Option<Self::Item> {
```

Comment:
not technically a typo (both work fine) but fair enough

Reply:
Yeah, definitely the "same". It just happened to be the only situation I found where it was styled as `fn next(&mut self) -> Option<T>` instead of `Option<Self::Item>`. This is a bit more style than typo I suppose. Happy to remove this change as well if you'd prefer!